tests/jax/ops/test_broadcast.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import jax.numpy as jnp
+import pytest
+from infra import run_op_test_with_random_inputs
+
+# TODO this skip reason is from old infra. Currently it fails with
+# `error: type of return operand 0 ('tensor<2x1xf32>') doesn't match function result type ('tensor<2x4xf32>') in function @main`.
+# Check why this happens and what is the state of this test in old infra (skip reason is old).
```

Comment:
Hm, how did pre-commit script allow this comment to be over 80 characters O.o If you are not sure too, please open new issue for us to investigate this later.